### DISCUSSION TITLE
spelling mistake of scss didn't show any error in development , only show during deployment

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [vercel/next.js#46593] [Help] [answered]
I was importing scss to my component . I wrote

tabs.module.scss instead of Tabs.module.scss . It should show error in vs code . What I need to do in vs code .

**The error also didn't show when next build**

![Screenshot (27)](https://user-images.githubusercontent.com/71011043/222047199-4b0021f8-0639-4ff7-ba88-0d9af03800a3.png)

**Comment by icyJoseph** [comment, marked as answer]
This is because of case sensitivity. May I ask what OS do you use to develop? 

It's like in MacOS, `Foo.txt` and `foo.txt` are the same file, but in Linux these are different files. The Vercel deployment environment is most likely a Linux distro, so it actually did look for Tabs.module.css, but if your local development is done in MacOS, the file system sees `Tabs.module.css` and `tab.module.css` as the same.

**Reply by thanhtutzaw** [reply]
It is Window OS .

**Reply by icyJoseph** [reply]
That explains, window is case insensitive, for windows `Foo.txt` and `foo.txt` are the same file.
